src/plugins/telemetryTable/TelemetryTableColumn.js

```diff
@@ -56,6 +56,14 @@ define(function () {
                 return formattedValue;
             }
         }
+
+        getParsedValue(telemetryDatum) {
+            if (telemetryDatum === undefined) {
```

Comment:
This works reasonably well for strings, but has problems for sort when the type is not a String (eg. `'' > 0 === false` and `'' < 0 === false`). I think `undefined` handling should be left to the formatters, but in every case I can think of they should just return `undefined` anyway :) So let's just pass undefined through to the formatters and fix any issues in there.

Returning undefined here has the same problem with sort (`undefined < 0 === false` and `undefined > 0 === false`). Depending on the sort algorithm being used, this may end up with all of the undefined values grouped at one end of the table, or they may end up being randomly scattered around. I'm ok with this for now. Let's not try to solve this problem right now, as the solution will depend on the data type (how do you handle an 'undefined' `Number`? Is it `Number.NEGATIVE_INFINITY`? How do we handle an `undefined` Date? etc.). Moreover, we don't have a great way of determining the correct JavaScript representation of a telemetry point's data values right now. I am increasingly thinking that the telemetry metadata should tell us this.

Reply:
agree, took the check out. I will just have to make sure the formatters in vista are able to handle undefined